tree/ntuple/test/ntuple_type_name.cxx

```diff
@@ -257,7 +257,7 @@ TEST(RNTuple, ContextDependentTypeNames)
       {
          const auto &fdesc = desc.GetFieldDescriptor(desc.FindFieldId("m", fooId));
          EXPECT_EQ(fdesc.GetTypeName(), "std::vector<std::int32_t>");
-         EXPECT_EQ(fdesc.GetTypeAlias(), "MyVec<std::int32_t>");
+         EXPECT_EQ(fdesc.GetTypeAlias(), "CustomStruct::MyVec<std::int32_t>");
```

Comment:
The old version may be correct, depending on how you argue: it's the type alias as the user wrote it, inside of `CustomStruct` which happens to have a `typedef`...